python/ray/services.py

```diff
@@ -785,18 +790,18 @@ def start_ray_processes(address_info=None,
   # warning messages when it starts up. Instead of suppressing the output, we
   # should address the warnings.
   redis_address = address_info.get("redis_address")
+  redis_shards = []
   if include_redis:
     redis_stdout_file, redis_stderr_file = new_log_files("redis",
                                                          redirect_output)
     if redis_address is None:
-      # Start a Redis server. The start_redis method will choose a random port.
+      # Start primary Redis. The start_redis method will choose a random port.
       redis_port, _ = start_redis(node_ip_address,
                                   stdout_file=redis_stdout_file,
```

Comment:
This reuses the same log file for all redis shards. We probably want separate ones, right?